app/Http/Transformers/SelectlistTransformer.php

```diff
@@ -33,7 +33,7 @@ public function transformSelectlist (LengthAwarePaginator $select_items)
         }
 
         $results = [
-            'items' => $items_array,
```

Comment:
@uberbrady can we switch this back to `items` here too, for the benefit of consistency in the API?

Reply:
yup!